src/amo/components/Search/index.js

```diff
@@ -120,10 +121,19 @@ export class SearchBase extends React.Component {
       <SearchSort filters={filters} pathname={pathname} />
     ) : null;
 
+    if (!queryParams.q) {
+      return (
+        <div className="Search">
+          <Card className="SearchContextCard">
+            <h1 className="SearchContextCard-header">Enter a search term and try again.</h1>
```

Comment:
Shouldn't we use `i18n` here?

Here is an example: https://github.com/mozilla/addons-frontend/blob/93548677fa52daabf950c6f947c89a5f6867429e/src/disco/components/Footer/index.js#L26